src/renderer/components/input/input.tsx

```diff
@@ -353,6 +353,7 @@ export class Input extends React.Component<InputProps, State> {
       dirty: _dirty, // excluded from passing to input-element
```

Comment:
Note: This was done to get rid of warnings that `blurOnEnter` was passed to DOM.